File: playbook-java/src/main/java/io/infinitic/playbook/java/Worker.java
package io.infinitic.playbook.java;

import io.infinitic.workers.InfiniticWorker;

import java.time.format.DateTimeFormatter;

public class Worker {
    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("hh:mm:ss:SSS");

    public static void main(String[] args) {
        try (InfiniticWorker worker = InfiniticWorker.fromConfigResource("/infinitic.yml")) {
            worker.start();
        }
    }
}

File: playbook-kotlin/src/main/kotlin/io/infinitic/playbook/kotlin/Worker.kt
package io.infinitic.playbook.kotlin

import io.infinitic.workers.InfiniticWorker
import java.time.format.DateTimeFormatter

internal val formatter = DateTimeFormatter.ofPattern("hh:mm:ss:SSS")

fun main() {
    InfiniticWorker.fromConfigResource("/infinitic.yml").use { worker ->
        worker.start()
    }
}
